src/bun.js/bindings/bindings.cpp

```diff
@@ -229,6 +229,45 @@ AsymmetricMatcherResult matchAsymmetricMatcher(JSGlobalObject* globalObject, JSC
             }
         }
 
+        return AsymmetricMatcherResult::FAIL;
+    } else if (auto* expectArrayContaining = jsDynamicCast<JSExpectArrayContaining*>(matcherPropCell)) {
+        JSValue expectedArrayValue = expectArrayContaining->m_arrayValue.get();
+
+        if (otherProp.isCell() && otherProp.asCell()->type() == ArrayType) {
+            if (expectedArrayValue.isCell() && expectedArrayValue.asCell()->type() == ArrayType) {
+                JSArray* expectedArray = jsDynamicCast<JSArray*>(expectedArrayValue);
+                JSArray* otherArray = jsDynamicCast<JSArray*>(otherProp);
+
+                unsigned expectedLength = expectedArray->length();
+                unsigned otherLength = otherArray->length();
+
+                // A empty array is all array's subset
+                if (expectedLength == 0) {
+                    return AsymmetricMatcherResult::PASS;
+                }
+
+                // O(m*n) but works for now
+                for (unsigned m = 0; m < expectedLength; m++) {
+                    JSValue expectedValue = expectedArray->get(globalObject, m);
```

Comment:
Instead of calling `get`, we should call `getIndex`

```suggestion
                    JSValue expectedValue = expectedArray->getIndex(globalObject, m);
```